pyproject.toml

```diff
@@ -35,6 +35,10 @@ torch = {version = "^2.1.0", optional = true}
 scikit-learn = {version = "^1.3.2", optional = true}
 umap-learn = {version = "^0.5.5", optional = true}
 tabulate = "^0.9.0"
+jax = "~0.4.30"
+jaxlib = "~0.4.30"
+tensorflow-io-gcs-filesystem = "0.29.0"
```

Comment:
I was not able to install due to this requirement:
```
ERROR: Could not find a version that satisfies the requirement tensorflow-io-gcs-filesystem==0.29.0 (from aws-fortuna) (from versions: 0.34.0, 0.36.0, 0.37.0)
ERROR: No matching distribution found for tensorflow-io-gcs-filesystem==0.29.0
```

Reply:
can you share your OS and Python version?

Reply:
MacOS and Python 3.11.9

Reply:
Thanks for pointing this out, I was able to reproduce this. This is happening with `pip`, right?